code/workspaces/client-api/src/config/graphql.js

```diff
@@ -1,37 +1,23 @@
-import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
+import { ApolloServer } from 'apollo-server-express';
 import { importSchema } from 'graphql-import';
 import { join } from 'path';
-import { makeExecutableSchema } from 'graphql-tools';
-import { authorise, verifyToken } from '../auth/authMiddleware';
-import status from '../status';
+
 import resolvers from '../schema/resolvers';
 
 function configureGraphQL(app) {
   const typeDefs = importSchema(join(__dirname, '..', 'schema', 'schema.graphql'));
-  const schema = makeExecutableSchema({
+
+  const server = new ApolloServer({
     typeDefs,
     resolvers,
+    context: ({ req }) => ({
+      user: req.user,
+      token: req.headers.authorization,
+    }),
+    playground: process.env.NODE_ENV !== 'production',
   });
 
-  const api = graphqlExpress(request => ({
-    schema,
-    context: {
-      user: request.user,
-      token: request.headers.authorization,
-    },
-  }));
-
-  const graphiql = graphiqlExpress({
-    endpointURL: '/graphiqlApi',
-  });
-
-  app.use('/api', authorise, verifyToken, api);
-  app.get('/status', status.get);
```

Comment:
Do we lose `/status` route with this update?

Reply:
No, it got moved to the base server